src/TodoApp/ApiModule.cs

```diff
@@ -52,7 +52,7 @@ public static IEndpointRouteBuilder MapTodoApiRoutes(this IEndpointRouteBuilder
             ITodoService service,
             CancellationToken cancellationToken) =>
             {
-                if (model is null || string.IsNullOrWhiteSpace(model.Text))
+                if (string.IsNullOrWhiteSpace(model.Text))
```

Comment:
This is new in rc1, we use the nullability of the parameter to determine if we should call your method with null. https://github.com/dotnet/aspnetcore/issues/32375

Reply:
Lots of nice little QoL and usability stuff coming in now 😎